crates/client/assets/schema.sdl

```diff
@@ -111,10 +111,6 @@ type Coin {
 	assetId: AssetId!
 	maturity: U64!
 	"""
-	The spendable status of the coin
-	"""
-	status: CoinStatus!
```

Comment:
From an API perspective, we still need to return the status of the coins and messages. Because we plan to move this API to the indexer and they can fetch this information. It is why I said to remove it only from "Compressed" types=)

Reply:
Hmm maybe, although no one was using this part of the API and so it's possible the indexer may not need to track spent coins either. I'd rather rip the bandaid fully off now, and add the status back later if there's an actual requirement for it.